main.cpp

```diff
@@ -108,9 +108,50 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct Person
+{
+	int age;
+	int height;
+	float hairLength;
+	float GPA;
+	unsigned int SATScore;
+	int distanceTraveled;
```

Comment:
Fix this indentation: see the course coding standard in the Readme.md
- [ ] revise